src/module/data/helpers.mjs

```diff
@@ -6,8 +6,8 @@ const {NumberField, SchemaField} = foundry.data.fields;
  * @returns A Schema with a value and max
  */
 export const barAttribute = (initial = 0) => new SchemaField({
-  value: new NumberField({initial}),
-  max: new NumberField({initial})
+  value: new NumberField({initial, nullable: false}),
+  max: new NumberField({initial, nullable: false})
```

Comment:
`integer: true`? Presumably these should have a minimum as well

Reply:
Realizing I'm only using this twice; Stamina *can* go negative, while Recoveries can't. I'm not sure how many other times we'll be using this helper, maybe I should just declare the properties and call it good.